x-pack/filebeat/input/awss3/s3_objects.go

```diff
@@ -448,7 +452,9 @@ func (p *s3ObjectProcessor) createEvent(message string, offset int64) beat.Event
 			},
 		},
 	}
-	event.SetID(objectID(p.s3ObjHash, offset))
+	if offset >= 0 {
```

Comment:
I like this approach. I think it would be beneficial to also omit `log.offset` when the `offset` is negative.

Reply:
I'll do that.